.github/workflows/contract-tests.yml

```diff
@@ -125,7 +129,7 @@ jobs:
 
       - uses: OffchainLabs/actions/run-nitro-test-node@main
         with:
-          nitro-testnode-ref: deneb-integration
+          nitro-testnode-ref: gzeoneth-patch-1
```

Comment:
why do we need this special testnode ref?

Reply:
testing stuff, almost there